examples/videocomposer/README.md

```diff
@@ -91,7 +106,7 @@ Please see [Project Page](https://videocomposer.github.io/) for more examples.
 
 We are searching for talented, motivated, and imaginative researchers to join our team. If you are interested, please don't hesitate to send us your resume via email yingya.zyy@alibaba-inc.com
 
-![figure1](assets/fig01.jpg "figure1")
+![figure1](https://raw.githubusercontent.com/wtomin/mindone-assets/main/videocomposer-assets/fig01.jpg "figure1")
```

Comment:
Thanks, it's a good improvement for simpler cloning. One thing to note, I think it's better to use `https://github.com/mindspore-lab/mindone/assets` domain. It can be done by editing any `.md` file inside MindONE repo and drag & dropping images there. GitHub will automatically generate links.